Traceback (most recent call last):
  File "iterator.py", line 6, in <module>
    g = my_generator()
  File "iterator.py", line 4, in my_generator
    return 1

WyppTypeError: 1

RÃ¼ckgabewert vom Typ `Iterator[int]` erwartet bei Aufruf der Funktion `my_generator`.
Aber der Aufruf gibt einen Wert vom Typ `int` zurÃ¼ck.

## Datei iterator.py
## RÃ¼ckgabetyp deklariert in Zeile 3:

def my_generator() -> [0;31m[1mIterator[int][0;0m:

## Fehlerhaftes return in Zeile 4:

    return 1

## Aufruf in Zeile 6 verursacht das fehlerhafte return:

g = [0;31m[1mmy_generator()[0;0m
